release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -313,6 +313,41 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
 [id="ocp-4-12-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+==== Updates to monitoring stack components and dependencies
+
+* Alertmanager to 0.24.0
```

Comment:
see from 4.11 release notes
https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-monitoring-updates-to-monitoring-stack-components-and-dependencies
4.11 Alertmanager is also 0.24.0, since the version is not updated, no need to mention Alertmanager here

Reply:
yeah I agree this sounds strange here. Maybe its worth mentioning somewhere that it stays at 0.24?

Reply:
I'll remove this from the list. We only need to mention the components with changed versions.